samples/tab-mail/ts/Manifest/manifest.json

```diff
@@ -0,0 +1,44 @@
+{
+  "$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.16/MicrosoftTeams.schema.json",
+  "manifestVersion": "1.16",
+  "version": "1.0.0",
+  "id": "{{Manifest-Id}}",
```

Comment:
Will it be an app id?

Reply:
only guid we were using, app id not required

Reply:
Then it should be documented in readme that replace this placeholder with any Guid or with your app id.